src/careamics/prediction_utils/prediction_outputs.py

```diff
@@ -105,13 +147,31 @@ def _combine_tiled_batches(
     tuple of (list of numpy.ndarray, list of TileInformation)
         Combined batches.
     """
-    # turn list of lists into single list
-    tile_infos = [
-        tile_info for *_, tile_info_list in predictions for tile_info in tile_info_list
-    ]
-    prediction_tiles: list[NDArray] = _combine_array_batches(
-        [preds for preds, *_ in predictions]
-    )
+    # Flatten predictions and tile_infos
```

Comment:
Could you explain why this had to change?

I don't see this function being called in `convert_outputs_microsplit` and I don't see why it should change  in this PR. It is difficult to review without a reason for its change.

Reply:
I still see it